pkgs/development/tools/electron/wrapper.nix

```diff
@@ -2,16 +2,25 @@
 , electron-unwrapped
 , wrapGAppsHook
 , makeWrapper
+, gsettings-desktop-schemas
+, glib
+, gtk3
+, gtk4
 }:
 
 stdenv.mkDerivation {
   pname = "electron";
   inherit (electron-unwrapped) version;
 
   nativeBuildInputs = [ wrapGAppsHook makeWrapper ];
+  buildInputs = [
+    # needed for GSETTINGS_SCHEMAS_PATH
+    gsettings-desktop-schemas glib gtk3 gtk4
```

Comment:
I am curious if we could remove either gtk 3 or 4. Maybe this is conditional based on the version?

Reply:
I copied that line from chromium/default.nix